target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -433,6 +433,9 @@ xiaomi,mir3p)
 	ucidef_set_led_switch "lan1-amber" "LAN1 (amber)" "$boardname:amber:lan1" "switch0" "0x02" "0x08"
 	ucidef_set_led_switch "lan2-amber" "LAN2 (amber)" "$boardname:amber:lan2" "switch0" "0x04" "0x08"
 	ucidef_set_led_switch "lan3-amber" "LAN3 (amber)" "$boardname:amber:lan3" "switch0" "0x08" "0x08"
+	;;	
+xiaoyu,xy-c5)
+	ucidef_set_led_default "sys" "sys" "$boardname:green:sys" "1"
```

Comment:
This setting is superfluous. `led_sys` will be always on after booting is completed

Reply:
@dengqf6 Won't you need a default-state = "on" for this in DTS?

Reply:
@adrianschmutzler no. It's set with /etc/init.d/done
AlthoughI 'd prefer to add a default state in dts anyway.

Reply:
Okay. But default-state = "on" will light it earlier than the userspace script, which starts blinking and then solid "on" again at the end?

Reply:
Ah, I overlooked that there is a separate power LED.

Reply:
Oh I just wait not enough long time for the system led stop blink. I delete this setting.